### PR TITLE
chore(tests): add TS support for e2e

### PR DESCRIPTION
**Pull request by filipesilva**
This PR enables angular.io's e2e test suite to be written in Typescript, and updates all `e2e-spec.js` to `e2e-spec.ts`. It doesn't update their code though, but rather simply renames the extension.

### CODE REVIEW

File: public/docs/_examples/cb-dependency-injection/e2e-spec.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-describe('Dependency Injection Cookbook', function () {
+describe('Dependency Injection Cookbookaaa', function () {
```
Comment (Foxandxss) [inline public/docs/_examples/cb-dependency-injection/e2e-spec.ts]: whaaat :P

Comment (filipesilva) [inline public/docs/_examples/cb-dependency-injection/e2e-spec.ts]: my baaad :D

**Comment by wardbell** [comment]
I'm excited to have this. But unfortunately,  it breaks VS Code intellisense for ​*everything*​, not just within the `e2e.spec.ts` files.  As soon as I roll back to master, intellisense returns.  Do you see that on your machine?

**Comment by filipesilva** [comment]
Closed in favour of #1556

**Pull request closed without merging**